frontend/src/metabase/dashboard/components/DashboardHeader/buttons/AddSectionButton.tsx

```diff
@@ -31,25 +30,25 @@ export const AddSectionButton = () => {
   return (
     <Menu position="bottom-end">
       <Menu.Target>
-        <span>
-          <Tooltip label={t`Add section`}>
-            <DashboardHeaderButton aria-label={t`Add section`}>
-              <Icon name="section" size={18} />
-            </DashboardHeaderButton>
-          </Tooltip>
-        </span>
+        <ToolbarButton
+          tooltipLabel={t`Add section`}
+          aria-label={t`Add section`}
+          icon="section"
+        />
       </Menu.Target>
       <Menu.Dropdown miw="100px">
         <Flex direction="column" align="center" gap="md" p="12px">
           {layoutOptions.map(layout => (
-            <SectionMenuItem
+            <Menu.Item
+              className={CS.bgBrandHover}
               key={layout.id}
+              bg={darken(color("bg-medium"), 0.1)}
```

Comment:
CSS functions don't need `color`, you can pass the color name directly like so
    ```suggestion
                  bg={darken("bg-medium", 0.1)}
    ```

I'd like you to use a CSS custom property, but I couldn't mix it to be close to what this color is, so just to be safe let's keep using `darken` for now.